vb_accumulator/src/universal.rs

```diff
@@ -133,10 +133,41 @@ impl<E> UniversalAccumulator<E>
 where
     E: PairingEngine,
 {
-    /// Create a new universal accumulator. Given the max size, it generates (max_size+1) initial elements
-    /// and adds them to the accumulator; these initial elements can never be removed or added back to the
-    /// accumulator
+    /// Create a new universal accumulator. Given the max size, it generates `max_size-n+1` initial elements
+    /// as mentioned in the paper. `n` elements are passed ar argument `xs`. These are generated
+    /// once for each curve as they depend on the order of the scalar field and then reused.
+    /// Constants for some of the curves are defined in `universal_init_constants.rs` are are computed
```

Comment:
Wrote "are" twice at the end.
Also I am not certain what this is referring to. If we provide some of the initialising constants in another file, then we have to make sure to initialise with more elements to account for the known elements

Reply:
Wanted to write "and are".
The other (than constants from sage file) elements are randomly generated in the for loop